### PR TITLE
Update BBEs of Email and UDP modules

### PR DESCRIPTION
**Pull request by Maninda**
## Purpose
Resolves https://github.com/ballerina-platform/ballerina-standard-library/issues/1355

### CODE REVIEW

File: examples/receive-email-using-client/receive_email_using_client.bal [hunk 1]
```diff
@@ -25,13 +25,13 @@ public function main() returns error? {
     // Closes the POP3 store which would close the TCP connection.
```
Comment (praneesha) [inline examples/receive-email-using-client/receive_email_using_client.bal:25]:
```suggestion
    // Closes the POP3 store, which would close the TCP connection.
```

File: examples/udp-client/udp_client.bal [hunk 2]
```diff
@@ -3,7 +3,7 @@ import ballerina/udp;
 
 public function main() returns error? {
   
-    // Create a new connectionless UDP client.
+    // Creates a new connectionless UDP client.
     // Optionally, you can provide the address that the socket needs to bind 
     // and the timeout in milliseconds, which specifies the read timeout value.
     // udp:Client client = new (localHost = "localhost", timeout = 5);
```
Comment (praneesha) [inline examples/udp-client/udp_client.bal:9]:
```suggestion
    // udp:Client client = new (localHost = "localhost", timeout = 5);
```
Why is this code commented?

File: examples/udp-client/udp_client.bal [hunk 3]
```diff
@@ -16,16 +16,16 @@ public function main() returns error? {
         data : msg.toBytes()
     };
 
-    // Send data to the remote host.
-    // The parameter is a Datagram record, which contains the `remoteHost`, `remotePort`,
-    // and the `data` to be sent.
+    // Sends data to the remote host.
```
Comment (praneesha) [inline examples/udp-client/udp_client.bal:19]:
```suggestion
    // Sends the data to the remote host.
```

File: examples/udp-connect-client/udp_connect_client.bal [hunk 4]
```diff
@@ -14,15 +14,15 @@ public function main() returns error? {
 
     string msg = "Hello Ballerina echo";
 
-    // Send data to the connected remote host.
+    // Sends data to the connected remote host.
```
Comment (praneesha) [inline examples/udp-connect-client/udp_connect_client.bal:17]:
```suggestion
    // Sends the data to the connected remote host.
```

File: examples/udp-listener/udp_listener.bal [hunk 5]
```diff
@@ -1,20 +1,20 @@
 import ballerina/io;
 import ballerina/udp;
 
-// Bind the service to the port.
+// Binds the service to the port.
 // Optionally, you can provide the `remoteHost` and `remotePort` to
 // configure the listener as a connected listener, which only
 // reads and writes to the configured remote host.
 // udp:Listener(8080, remoteHost = "www.remote-clinet.com", remotePort = 9090)
```
Comment (praneesha) [inline examples/udp-listener/udp_listener.bal:8]:
```suggestion
// udp:Listener(8080, remoteHost = "www.remote-clinet.com", remotePort = 9090)
```
Why is this code commented out.

File: examples/udp-listener/udp_listener.bal [hunk 6]
```diff
@@ -1,20 +1,20 @@
...
-    // You may replace the `onBytes` method with `onDatagram`, which reads the data as
-    // readonly & udp:Datagram.
+    // This remote method is invoked once the content is received from the
+    // client. You may replace the `onBytes` method with `onDatagram`, which
+    // reads the data as `readonly & udp:Datagram`.
     remote function onDatagram(readonly & udp:Datagram datagram) 
         returns udp:Datagram|udp:Error? {
         io:println("Received by listener: ", string:fromBytes(datagram.data));
-        // Echo back the data to the same client.
+        // Echos back the data to the same client.
```
Comment (praneesha) [inline examples/udp-listener/udp_listener.bal:17]:
```suggestion
        // Echoes back the data to the same client.
```

File: examples/udp-client/udp_client.bal (hunk 2 above)
Comment (Maninda) [reply examples/udp-client/udp_client.bal:9]: This is given as a sample code.

File: examples/udp-listener/udp_listener.bal (hunk 5 above)
Comment (Maninda) [reply examples/udp-listener/udp_listener.bal:8]: This is given as a sample code.

**Review by praneesha — APPROVED** [review]

**Review by MadhukaHarith92 — APPROVED** [review]
LGTM

**Pull request merged**